torch/quantization/fx/convert.py

```diff
@@ -180,7 +181,7 @@ def convert(model: GraphModule, is_reference: bool = False,
         custom_module_classes=custom_module_classes)
 
     quantized_graph = Graph()
-    env: Dict[str, Tuple[Node, Optional[torch.dtype]]] = {}
+    env: Dict[str, Dict[Optional[torch.dtype], Node]] = defaultdict(lambda: defaultdict(Node))
```

Comment:
qq, what's the motivation for Dict[str, Dict[dtype, Node]] versus Dict[Tuple[str, dtype], Node]?

Reply:
Oh I think they are pretty similar, I don't really have a strong preference here, maybe Dict[str, Dict[dtype, Node]] is slightly easier to manipulate compared to the other. Is there a reason that makes one better than another?

Reply:
I don't feel strongly, current code lg